content/grants/bounty-calendar-ui.md

```diff
@@ -0,0 +1,55 @@
++++
+title = "Calendar UI"
+date = 2021-01-01
+[taxonomies]
+grant_type = ["bounties"]
+grant_category = ["App Dev: Other"]
+[extra]
+image = ""
+description = "Develop a UI for improved access to the Calendar Gall app"
+reward = 5
+assignee = "~fabnev-hinmur"
+id = "B0014"
```

Comment:
I think the ID maps to something @tylershuster implemented for the old grants site. Might want to double check the template to make sure this doesn't create any weird links.